projects/power_distribution/src/pd_fan_ctrl.c

```diff
@@ -56,7 +57,18 @@ static void prv_front_temp_to_fan_percent(uint16_t v_measured, uint8_t *fan_spee
     *fan_speed = 100;
     return;
   }
-  double ratio = s_fan_storage.ref_reading / 100.0;
+  if (v_measured == 0) {
+    // just to be safe to avoid divide-by-zero errors
+    *fan_speed = 0;
+    return;
+  }
+
+  if (v_measured > (FRONT_FAN_CTRL_MAX_VALUE_MV + ADC_EPSILON_MV)) {
```

Comment:
Maybe we could check for both these conditions in line 60, eg
`if` (v_measured == 0 || v_measured > (FRONT_FAN_CTRL_MAX_VALUE_MV + ADC_EPSILON_MV)){`

Reply:
We could do that